content/utmt/rooms/creating new rooms/_index.md

```diff
@@ -8,15 +8,39 @@ resources:
   - name: optimalRoomSide
     src: "optimalRoomSide.png"
     title: Optimal placing for left and right transitions
+  - name: RoomSetup
+    src: "RoomSetup.png"
+    title: This is how it should look now
+  - name: RoomOnCreate
+    src: "RoomOnCreate.png"
+    title: This is how it will look when first adding a new room
+  - name: ViewsSetup
+    src: "ViewsSetup.png"
+    title: If you've done everything correctly this is how it should look 
 ---
```

Comment:
Personally, I like camelCase for assets, so `roomOnCreate` instead of `RoomOnCreate`. For everyones sake who wants to contribute, the assets should all have a similar naming style.